scss/boosted-grid.scss

```diff
@@ -66,5 +69,10 @@ $utilities: map-get-multiple(
     "padding-end",
     "padding-bottom",
     "padding-start",
+    "gap",
+    "row-gap",
+    "column-gap",
```

Comment:
A PR was made on Bootstrap's side for this part.
Currently waiting for their answer...

Reply:
See https://github.com/twbs/bootstrap/pull/39068

Reply:
Can I drop this modification so that we can merge this PR? And if it's accepted upstream, it'll be integrated when the Bootstrap commits will be merged into Boosted?